frontend/src/scenes/project-homepage/projectHomepageLogic.tsx

```diff
@@ -20,16 +33,61 @@ export const projectHomepageLogic = kea<projectHomepageLogicType>({
         ],
     },
 
+    reducers: () => ({
+        sessionRecordingId: [
+            null as null | string,
+            {
+                openRecordingModal: (_, { sessionRecordingId }) => sessionRecordingId,
+                closeRecordingModal: () => null,
+            },
+        ],
+    }),
+
+    loaders: ({ values }) => ({
+        recordings: [
+            [] as SessionRecordingType[],
+            {
+                loadRecordings: async (_, breakpoint) => {
+                    const paramsDict = {
+                        limit: 5,
+                        session_recording_duration: DEFAULT_DURATION_FILTER,
+                    }
+                    const response = (await api.get(
+                        `api/projects/${values.currentTeamId}/session_recordings?${toParams(paramsDict)}`
+                    )) as SessionRecordingsResponse
+
+                    breakpoint()
+                    return response.results
+                },
+            },
+        ],
+    }),
+
```

Comment:
Does it make sense to try to reuse sessionRecordingsTableLogic here? (I'm not up to date on how that logic work) Thinking about consistency, we're subscribing to dashboardLogic here and I'm assuming you're going to need to route a bunch more logic through dashboardLogic here for other home page widgets.

(Not a blocker but could keep clarity in the logics here)

Reply:
100% agree - just moved it to using `sessionRecordingsTableLogic`. Feels much better!